contributors/devel/sig-node/updating-pause-images.md

```diff
@@ -0,0 +1,30 @@
+# Updating Pause Image References
+
+The Kubernetes `pause` image that is used to run a 'parent' container for all
+the other containers inside your pod. This image is referenced across many
+projects within the Kubernetes ecosystem, and it's important that we keep them
+all up to date to ensure stable platform support and to avoid confusing things
```

Comment:
whether we can classify these projects to several levels, for projects like `minikube` and `kind` don't depend on each other.  Projects in same level has no update order at all. e.g. 
`first level`: k/k, k/test-infra
`second level`: k-sigs, like image-builder, kind
`third level`: runtime and other related projects.

since I'm not quite familiar with the dependence between these projects, forget me if not proper.